superduper/base/datalayer.py

```diff
@@ -419,6 +418,20 @@ def _insert(
 
         return inserted_ids, None
 
+    def _auto_create_table(self, table_name, documents):
```

Comment:
if create table is compulsory 
why auto create?

Reply:
After modification, it has become two choices:

1. Explicitly manually creating the table
2. Implicitly automatically creating the table

In both cases, the table will be created regardless, and it is unrelated to the backend.